Formula/gf.rb

```diff
@@ -0,0 +1,23 @@
+class Gf < Formula
+  desc "GoFrame is a modular, powerful, high-performance app dev framework of Golang"
+  homepage "https://goframe.org"
+  url "https://github.com/gogf/gf/archive/refs/tags/v2.2.2.tar.gz"
+  sha256 "6c289efa8ec11a035d350c61788ae041d74b0e61142bec87ea3461f01c16f88e"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cmd/gf" do
+      system "go", "build", *std_go_args(ldflags: "-s -w")
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/gf --version 2>&1")
+    assert_match "GoFrame CLI Tool v#{version}, https://goframe.org", output
+    assert_match "GoFrame Version: cannot find go.mod", output
+    assert_match "CLI Installed At: #{HOMEBREW_PREFIX}/Cellar/gf/#{version}/bin/gf", output
+    assert_match "Current is a custom installed version, no installation information.", output
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
I add a init command of this tool. and other command requires more input, should be verified by user.